src/app/game/settings/account/change/email/complete/page.tsx

```diff
@@ -0,0 +1,48 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import Link from "next/link";
+import Image from "next/image";
+
+export default function AuthResetPasswordComplete() {
```

Comment:
パス名に基づいて命名規則を定めます。
以下の様に修正依頼とします！
```suggestion
export default function GameSettingsAccountChangeEmailComplete() {
```